sdk/communication/communication-sms/recordings/browsers/smsclient/recording_sends_a_sms_message_to_multiple_recipients.json

```diff
@@ -0,0 +1,22 @@
+{
+ "recordings": [
+  {
+   "method": "POST",
+   "url": "https://endpoint/sms",
+   "query": {
+    "api-version": "2021-03-07"
+   },
+   "requestBody": "{\"from\":\"+18005551234\",\"smsRecipients\":[{\"to\":\"+18005551234\",\"repeatabilityRequestId\":\"sanitized\",\"repeatabilityFirstSent\":\"Thu, 01 Jan 1970 00:00:00 GMT\"},{\"to\":\"+18332321226444\",\"repeatabilityRequestId\":\"sanitized\",\"repeatabilityFirstSent\":\"Thu, 01 Jan 1970 00:00:00 GMT\"}],\"message\":\"test message\",\"smsSendOptions\":{\"enableDeliveryReport\":true,\"tag\":\"SMS_LIVE_TEST\"}}",
```

Comment:
I believe these numbers should be redacted.

Reply:
Actually, these phone numbers aren't real numbers, one is actually a dummy number put in by the recorder, the other is invalid (too long)

Reply:
I see there is a to number that looks like it's real +18332321226444. 
In C# though, I just redacted all the From and To numbers regardless if they are real or fake. I found that was the easiest way to make sure our real numbers are protected.